dozer/db.py

```diff
@@ -136,24 +136,16 @@ def __repr__(self):
             return await conn.fetch(f"""SELECT * FROM {cls.__tablename__};""")
 
     @classmethod
-    async def delete(cls, data_column, data):
-        """Deletes by one criteria"""
+    async def delete(cls, data_tuple_list):
+        """Deletes by any number of criteria specified as a list of (data_column, data) tuples"""
         async with Pool.acquire() as conn:
             statement = f"""
             DELETE FROM  {cls.__tablename__}
-            WHERE {data_column} = {data};
-            """
-            print(statement)
-            await conn.execute(statement)
-
-    @classmethod
-    async def dual_criteria_delete(cls, data_column, data, data_column_two, data_two):
-        """Deletes by two criteria"""
-        async with Pool.acquire() as conn:
-            statement = f"""
-            DELETE FROM  {cls.__tablename__}
-            WHERE {data_column} = {data} AND {data_column_two} = {data_two};
-            """
+            WHERE {data_tuple_list[0][0]} = {data_tuple_list[0][1]}"""
+            if len(data_tuple_list) > 1:
+                for i in range(1, len(data_tuple_list)):
+                    statement += f" AND {data_tuple_list[i][0]} = {data_tuple_list[i][1]}"
```

Comment:
First of all, I want to thank you for taking the time to work on this.

Second, unfortunately this is vulnerable to SQL injection. Using asyncpg query parameters would eliminate the vulnerability.

Reply:
Oops, my bad right there. I am quite new to SQL, thanks for showing that out.